reference/shipping.v3.yml

```diff
@@ -313,77 +313,46 @@ components:
 
         * For more information on Authenticating BigCommerce APIs, see: [Authentication](/api-docs/getting-started/authentication).
   schemas:
-    shippingMethod_Full:
-      title: shippingMethod_Full
-      allOf:
-        - type: object
-          properties:
-            id:
-              description: Shipping-method ID. READ-ONLY
-              example: 1
-              type: integer
-        - $ref: '#/components/schemas/shippingMethod_Base'
-      x-internal: false
-    shippingMethod_Base:
-      title: shippingMethod_Base
+    customsInformation_request:
+      title: customsInformationRequest
+      description: Data about the customs information object.
       type: object
+      x-internal: false
       properties:
-        name:
-          description: Display name for shipping method.
-          example: Flat Rate
+        product_id:
+          description: The id of the product which the customs information data will apply to.
+          type: integer
+          format: int32
+          example: 77
+        country_of_origin:
           type: string
-        type:
-          title: Shipping Method Type
-          example: perorder
+          description: 'The country of manufacture, production, or growth represented in ISO 3166-1 alpha-2 format.'
+          example: US
+        commodity_description:
+          description: Description that provides information for customs to identify and verify shapes physical characteristics and packaging of each shipment.
           type: string
+          minLength: 0
+          maxLength: 100
+          example: Baseball caps
```

Comment:
This question might be outside our purview, but do the commodity descriptions need to adhere to a specific standard like those listed here: https://dataweb.usitc.gov/classification/commodity-description/HTS/4? Disregard if NBD.

Reply:
Might be for a different ticket